backend/lucene/src/main/java/org/hibernate/search/backend/lucene/search/predicate/impl/LuceneNestedPredicate.java

```diff
@@ -57,10 +58,16 @@ public static Query createNestedQuery(String parentNestedDocumentPath, String ne
 	static class Builder extends AbstractBuilder implements NestedPredicateBuilder {
 		private LuceneSearchPredicate nestedPredicate;
 
+		private static List<String> extendPath(List<String> nestedPathHierarchy, String nextPath) {
+			List<String> result = new ArrayList<>( nestedPathHierarchy );
+			result.add( nextPath );
+			return result;
+		}
+
 		Builder(LuceneSearchContext searchContext, String absoluteFieldPath,
 				List<String> nestedPathHierarchy) {
 			// The given list includes absoluteFieldPath at the end, but here we don't want it to be included.
-			super( searchContext, absoluteFieldPath, nestedPathHierarchy.subList( 0, nestedPathHierarchy.size() - 1 ) );
+			super( searchContext, absoluteFieldPath, extendPath( nestedPathHierarchy, absoluteFieldPath ) );
```

Comment:
What is this change for exactly? It seems unrelated...

Reply:
So no PR related. But I tried to use with the nest () function, but it doesn't work properly. Only after this change does it become correct. This hotfix fixes this feature. Rather, it's a problem for a new submission.

Reply:
I'll remove it. However, the future will need to report a new problem.

Reply:
Yes, please report a new problem with a reproducer.